### ISSUE TITLE
Not able to keyboard Tab out of the range selector using Highcharts Stock module

### ISSUE THREAD

**Issue by hubertkozik** [highcharts/highcharts#17866]
#### Expected behaviour

Users should be able to navigate through the data points, chart menu, and other chart controls without any issues/loops.

#### Actual behaviour

If you navigate by pressing Tab to range selector, you won't be able to go out of the range selector, keyboard navigation will be stuck into buttons from the range selector.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/y03ge4sa/
1) Press the tab until you move into the range selector buttons.
2) Press the tab multiple times - you cannot go out of range selector section to e.g. series.

#### Product version
Highstock v10.2.1

**Comment by oysteinmoseng** [comment]
Closing, will be fixed in v11.
